### PR TITLE
pd: update default value of `max-merge-region-size`

### PR DESCRIPTION
**Pull request by HuSharp**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)

### What is the related PR or file link(s)?

related PR: https://github.com/tikv/pd/pull/8445

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by HuSharp** [comment]
@LykxSassinator PTAL, thx :)

**Review by lhy1024 — APPROVED** [review]

File: pd-configuration-file.md [hunk 1]
```diff
@@ -256,7 +256,7 @@ pd-server 相关配置项。
 ### `max-merge-region-size`
 
 + 控制 Region Merge 的 size 上限，当 Region Size 大于指定值时 PD 不会将其与相邻的 Region 合并。
-+ 默认：20
++ 默认：54
```
Comment (lhy1024) [inline pd-configuration-file.md:259]:
```suggestion
+ 默认：54 （8.2.0 及之前版本默认值为20）
```

**Review by rleungx — APPROVED** [review]

File: pd-configuration-file.md [hunk 2]
```diff
@@ -256,7 +256,7 @@ pd-server 相关配置项。
...
 + 单位：MiB
 
 ### `max-merge-region-keys`
```
Comment (lhy1024) [inline pd-configuration-file.md:262]: We also need to update it

**Review by lhy1024 — APPROVED** [review]

**Review by LykxSassinator — APPROVED** [review]
LGTM

File: releases/release-8.2.0.md [hunk 3]
```diff
@@ -183,6 +183,8 @@ TiDB 版本：8.2.0
 | TiKV | [`max-apply-unpersisted-log-limit`](/tikv-configuration-file.md#max-apply-unpersisted-log-limit-从-v810-版本开始引入) | 修改 | 默认值从 `0` 修改为 `1024`，代表允许 apply 已经 `commit` 但尚未持久化的 Raft 日志的最大数量为 1024，用于降低 TiKV 节点上因 I/O 抖动导致的长尾延迟。 |
 | TiKV | [`server.grpc-compression-type`](/tikv-configuration-file.md#grpc-compression-type) | 修改 | 该配置项现在也会影响 TiKV 向 TiDB 发送的响应消息的压缩算法。开启压缩可能消耗更多 CPU 资源。 |
 | TiFlash | [`security.redact_info_log`](/tiflash/tiflash-configuration.md#配置文件-tiflashtoml) | 修改 | 可选值新增 `marker` 选项。当配置项的值设置为 `marker` 时，日志中的用户数据会被标记符号 `‹ ›` 包裹。 |
+| PD | [`schedule.max-merge-region-size`](/pd-configuration-file.md#max-merge-region-size) | 修改 | 默认值从 `20` MiB 修改为 `54`。 |
```
Comment (qiancai) [inline releases/release-8.2.0.md:186]:
```suggestion
| PD | [`schedule.max-merge-region-size`](/pd-configuration-file.md#max-merge-region-size) | 修改 | 默认值从 `20` MiB 修改为 `54` MiB。 |
```

**Comment by HuSharp** [comment]
replace by https://github.com/pingcap/docs-cn/pull/18336

**Pull request closed without merging**